packages/handlersjs-http/lib/handlers/error.handler.ts

```diff
@@ -43,7 +44,7 @@ export class ErrorHandler extends Handler<HttpHandlerResponse, HttpHandlerRespon
 
   private createErrorResponse(res: HttpHandlerResponse, msg: string, showError: boolean) {
 
-    return showError ? { ...res, body: msg + ': ' + res.body } : { ...res, body: msg };
+    return showError ? { ...res, body: msg + ': ' + res.body, status: res.status } : { ...res, body: msg, status: res.status  };
```

Comment:
Is already part of the destructuring.
```suggestion
    return showError ? { ...res, body: msg + ': ' + res.body } : { ...res, body: msg };
```

Reply:
Yes this one useless.